vm/mx.vm/mx_vm.py

```diff
@@ -229,24 +234,39 @@ def _patch_darwin_jdk():
             vm_name = graalvm_vm_name(self, join(_jdk_dir, _src_jdk_base))
             if mx.get_os() == 'darwin':
                 # on macOS the <arch> directory is not used
-                _add(layout, "<jdk_base>/jre/lib/", "extracted-dependency:truffle:TRUFFLE_NFI_NATIVE/bin/<lib:trufflenfi>")
-                _add(layout, "<jdk_base>/jre/lib/server/vm.properties", "string:name=" + vm_name)
+                if _src_jdk_has_jre:
+                    _add(layout, "<jdk_base>/jre/lib/", "extracted-dependency:truffle:TRUFFLE_NFI_NATIVE/bin/<lib:trufflenfi>")
```

Comment:
To avoid duplication we can introduce `<jre_base>` similar to `<jdk_base>`

Reply:
done